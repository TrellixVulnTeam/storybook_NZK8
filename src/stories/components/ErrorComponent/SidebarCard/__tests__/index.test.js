import React from 'react';
import { mount } from 'enzyme';
import SidebarError from 'components/ErrorComponent/SidebarCard/index';

describe('<SidebarError />', () => {
  it('should render correctly with mandatory props', () => {
    const sut = mount(<SidebarError name="Name" />);

    expect(sut).toMatchSnapshot();
  });
});
