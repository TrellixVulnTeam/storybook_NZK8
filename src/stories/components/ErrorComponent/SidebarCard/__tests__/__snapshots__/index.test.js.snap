// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SidebarError /> should render correctly with mandatory props 1`] = `
<SidebarError
  name="Name"
>
  <styled__WrapperDiv>
    <div
      className="styled__WrapperDiv-sc-1re590x-1 kMGrIT"
    >
      <styled__IconWrapper>
        <div
          className="styled__IconWrapper-sc-1re590x-2 gaNzHI"
        >
          <styled__AlertIcon>
            <test-file-stub
              className="styled__AlertIcon-sc-1re590x-0 jeIbUB"
            />
          </styled__AlertIcon>
        </div>
      </styled__IconWrapper>
      <styled__ContentWrapper>
        <div
          className="styled__ContentWrapper-sc-1re590x-3 gXdxwg"
        >
          <styled__ErrorMessage>
            <p
              className="styled__ErrorMessage-sc-1re590x-4 kQcggX"
            >
              Something went wrong while loading
               Name
            </p>
          </styled__ErrorMessage>
          <styled__ErrorHelpMessage>
            <p
              className="styled__ErrorHelpMessage-sc-1re590x-5 bbQzEq"
            >
              Please try again in sometime
            </p>
          </styled__ErrorHelpMessage>
        </div>
      </styled__ContentWrapper>
    </div>
  </styled__WrapperDiv>
</SidebarError>
`;
